commons/src/main/kotlin/androidx/lifecycle/LiveDataHiddenApi.kt

```diff
@@ -3,6 +3,7 @@ package androidx.lifecycle
 /**
  * https://github.com/kxfeng/livedata-ext/blob/master/library/src/main/java/androidx/lifecycle/LiveDataHiddenApi.kt
  */
+@Suppress("UndocumentedPublicFunction")
```

Comment:
You don't need to suppress it if you've added the documentation right?

And add in actual documentation too. Explain why we're doing this weird package structure for this